src/lib.rs

```diff
@@ -131,15 +202,15 @@ fn raw_to_parse_error(map: &CodeMap, err: Error) -> Box<Error> {
 ///
 /// ```
 /// fn main() -> Result<(), Box<grass::Error>> {
-///     let sass = grass::from_path("input.scss")?;
+///     let sass = grass::from_path("input.scss", &grass::Options::default())?;
 ///     Ok(())
 /// }
 /// ```
 /// (grass does not currently allow files or paths that are not valid UTF-8)
 #[cfg_attr(feature = "profiling", inline(never))]
 #[cfg_attr(not(feature = "profiling"), inline)]
 #[cfg(not(feature = "wasm"))]
-pub fn from_path(p: &str) -> Result<String> {
+pub fn from_path(p: &str, options: &'_ Options<'_>) -> Result<String> {
```

Comment:
Is it an error to write `options: &Options<'_>`? `&'_` in that context should be redundant, unless I'm missing something.